r/R/dplyr-funcs-datetime.R

```diff
@@ -57,6 +57,7 @@ register_bindings_datetime_utility <- function() {
       tz <- Sys.timezone()
     }
 
+    # convert a "timezone-naive" into a "timezone-aware" timestamp
```

Comment:
Could you expand a little bit more here? What about these timestamps make them timezone-naive? And why is that important?

Reply:
Done. Hopefully that offers a bit more clarity.